Solar_radiation_scripts/retrieve_PSM_data.R

```diff
@@ -0,0 +1,52 @@
+# Code source: https://github.com/dazhiyang/SolarData
+# https://github.com/dazhiyang/SolarData/blob/master/examples/PSMExample.r
+
+# Install and load devtools package
+install.packages("devtools")
+
+# Install and load the SolMod package
+install_github("dazhiyang/SolarData")
+
+#load libraries
+library("devtools")
+library("SolarData")
+
+
+# set working directory
+dir_PSM <- "C:/Users/Hayley/Documents/USGS/R/delaware-water-temp"
+setwd(dir_PSM)
```

Comment:
this shouldn't be needed if you're using R projects. The relative file path should be set to the project name already.